redaxo/src/addons/mediapool/functions/function_rex_mediapool.php

```diff
@@ -9,48 +9,15 @@
 /**
  * Erstellt einen Filename der eindeutig ist für den Medienpool.
  *
- * @param string $FILENAME      Dateiname
+ * @param string $mediaName      Dateiname
  * @param bool   $doSubindexing
  *
  * @return string
+ * @deprecated
```

Comment:
```suggestion
 * @deprecated since 2.11, use `rex_mediapool::getUniqueMediaName` instead
```

Nach dem Schema bitte bei allen Funktionen.

Und in der package.yml dann schon mal die neue version `2.11.0-dev` setzen.